packages/ember-glimmer/lib/syntax/outlet.js

```diff
@@ -212,10 +216,19 @@ class OutletComponentManager {
   didUpdate(state) {}
 }
 
+runInDebug(() => {
+  OutletComponentManager.prototype._pushToDebugStack = function(definition, environment) {
```

Comment:
Thanks, I'll follow up in a subsequent PR and refactor away the duplicate `_pushToDebugStack`s